src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -317,7 +317,7 @@ protected internal void SetHtmlContent(string html)
 			);
 		}
 
-		internal static UIElement GetElementFromHandle(IntPtr handle)
+		internal static UIElement GetElementFromHandle(ObjCRuntime.NativeHandle handle)
```

Comment:
inside a `*.wasm.cs` file ?

Reply:
Yep, that was a find/replace gone bad ;)